sds_data_manager/lambda_code/SDSCode/indexer.py

```diff
@@ -21,6 +21,35 @@
 s3 = boto3.client("s3")
 
 
+def get_file_creation_date(s3_uri):
+    """Get s3 file creation date.
+
+    Parameters
+    ----------
+    s3_uri: str
+        S3 URI. Eg. s3://bucket/filepath/filename.ext
+
+    creation_date: datetime.datetime
+        Last modified data of s3 file.
+    """
+    # Create an S3 client
+    s3_client = boto3.client("s3")
+
+    # Retrieve the metadata of the object
+    bucket_name, key = s3_uri.replace("s3://", "").split("/", 1)
+    logger.info(f"bucket_name: {bucket_name}")
+    logger.info(f"key: {key}")
+
+    response = s3_client.head_object(Bucket=bucket_name, Key=key)
+    file_creation_date = response["LastModified"]
+
+    # time looks like this:
+    # 2024-01-25 23:35:26+00:00
+    # Formats the datetime object to a string with the format "%Y%m%d".
+    ingestion_data_str = file_creation_date.strftime("%Y%m%d")
+    return datetime.datetime.strptime(ingestion_data_str, "%Y%m%d")
```

Comment:
If we are using datetime objects in the DB, do we actually want to keep the entire datetime interval?